lib/event_listeners/foodEvents.js

```diff
@@ -0,0 +1,91 @@
+const $ = require('jquery')
+const Food = require('../models/Food')
+const Ajax = require('../ajax_requests/ajax')
+
+// event listener for adding food
+
+$(document).ready(function(){
+  $("#food_form").submit(event => {
+    let name = $("input[name=name]").val()
+    let calories = $("input[name=calories]").val()
+    if (name.length === 0) {
+      event.preventDefault();
+      $("p#submit-food-name").append("Please enter a food name.")
+      let k = $('p#submit-food-name').html();
+    } else if (calories.length === 0) {
+      event.preventDefault();
+      $("p#submit-calories").append("Please enter a calorie amount.")
+    } else {
+      $("p#submit-food-name").remove()
+      $("p#submit-calories").remove()
+      Ajax.postFoods(name, calories)
+      $('input').val('').removeAttr('checked').removeAttr('selected');
+    }
+  })
+})
+
+// event listener for deleting food
```

Comment:
I like this and was thinking bout doing that with meal events too!  nice thinking